backend/dataall/modules/datasets/cdk/dataset_stack.py

```diff
@@ -242,8 +259,7 @@ def __init__(self, scope, id, target_uri: str = None, **kwargs):
                     sid="ListDatasetBucket",
                     actions=[
                         "s3:ListBucket",
-                        "s3:GetBucketLocation",
-                        "s3:GetBucketAcl"
```

Comment:
why was this action removed?

Reply:
Coming from a merge conflict. Added it back.